src/System.IO.FileSystem/tests/FileStream/CanSeek.Windows.cs

```diff
@@ -1,25 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using Microsoft.Win32.SafeHandles;
-using System;
-using System.IO;
-using Xunit;
-
-namespace System.IO.FileSystem.Tests
-{
-    partial class FileStream_CanSeek
```

Comment:
Why delete?  I knew it was a windows only test, thus the name.  Can you just exclude it?